src/schema/v2/artist/verifiedRepresentatives.ts

```diff
@@ -0,0 +1,51 @@
+import { GraphQLFieldConfig, GraphQLList, GraphQLNonNull } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { PartnerType } from "schema/v2/partner/partner"
+
+const VerifiedRepresentatives: GraphQLFieldConfig<
+  { _id: string },
+  ResolverContext
+> = {
+  type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(PartnerType))),
+  resolve: async (
+    { _id },
+    options,
+    { verifiedRepresentativesLoader, partnersLoader }
+  ) => {
+    if (!verifiedRepresentativesLoader) {
+      throw new Error("You are not authorized to perform this action.")
+    }
+    try {
+      const verifiedRepresentatives = await verifiedRepresentativesLoader(
+        { artist_id: _id },
+        options
+      )
+
+      if (!verifiedRepresentatives || !Array.isArray(verifiedRepresentatives)) {
+        throw new Error("Invalid response from verifiedRepresentativesLoader.")
+      }
+
+      const partnerIds = verifiedRepresentatives.map(
+        ({ partner_id }) => partner_id
+      )
+
+      if (!Array.isArray(partnerIds) || partnerIds.length === 0) {
+        return []
+      }
+
+      const response = await partnersLoader({ ids: partnerIds })
```

Comment:
Is `ids` the right argument to pass? Looking in Gravity, seems like that endpoint uses `id` - https://github.com/artsy/gravity/blob/112ffe7907729535a320e14fdc1fe43f039b628f/app/api/v1/partners_endpoint.rb#L228

Reply:
💚